client/components/credit-card/style.scss

```diff
@@ -0,0 +1,23 @@
+@import './stored-card';
+
+div.credit-card {
+    border: 1px solid $gray-lighten-30;
+    border-top: none;
+    padding-left: 2px;
+    min-height: 50px;
+
+    &:first-of-type {
+        border-top: 1px solid $gray-lighten-30;
+    }
+
+    &.selectable:not(.selected) {
+        cursor: pointer;
+    }
+
+    &.selected {
+        cursor: default;
+        border-left: 3px solid $alert-green;
+        background-color: #fafdf6;
```

Comment:
Custom color here. @folletto any suggestions for a named color?

Reply:
I'd suggest to use a derivative color in cases like this: since the base is `$alert-green`, we can try with `lighten( $alert-green, 30 )` (I haven't run the repo to see if that matches, tweak the number as needed to get a closer match).